sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/GroupIntoBatches.java

```diff
@@ -221,6 +221,31 @@ private GroupIntoBatches(BatchingParams<InputT> params) {
             duration));
   }
 
+  public GroupIntoBatches<K, InputT> withByteSize(Long batchSizeBytes) {
```

Comment:
This probably deserves a javadoc comment. Why do we use big `Long` instead of `long`?

Reply:
Just misuse on my part, I thought I'd copied what was done in the other methods. Changed in a recent commit 👍